src/domain/session/room/timeline/tiles/SimpleTile.js

```diff
@@ -93,6 +94,17 @@ export class SimpleTile extends ViewModel {
 
     // update received for already included (falls within sort keys) entry
     updateEntry(entry, param) {
+        const replyEntry = entry.contextEntry;
+        if (replyEntry) {
+            // this is an update to contextEntry used for replyPreview
+            const action = this._replyTile?.updateEntry(replyEntry);
+            if (action?.shouldReplace) {
+                this._replyTile = this._tileCreator(replyEntry);
+            }
+            else {
+                this._replyTile?.emitChange();
+            }
+        }
```

Comment:
This logic should probably be moved to `BaseMessageTile.updateEntry()`. For one, `SimpleTile` does not know what a replyTile is (it's defined in `BaseMessageTile`). Also not all SimpleTiles are message tiles.